scripts/variantstore/beta_docs/gvs-outputs.md

```diff
@@ -1,14 +1,48 @@
 # Working with the outputs of the Genomic Variant Store
 
-## Applying the GVS Joint Calling Filters
-During joint calling using GVS, filters are evaluated at the site and allele level. All data is retained, but sites failing the site filters will be indicated by one of the site level filters from the table below appearing in the VCF FILTER field. Genotypes that have been filtered by the Genotype filters in the table below will be tagged in the FORMAT FT field. Variants that pass site level filters will have “.”.
+## Content in the VCF
+This is the full list of the filters, format, comment, and info fields in the GVS VCFs. Note that they are different from the ones in the GATK WARP Joint Calling VCFs. Additionally, some are new with the introduction of VETS. Read more about VETS [here](https://github.com/broadinstitute/gatk/blob/gvs_0.5.5/scripts/variantstore/docs/release_notes/VETS_Release.pdf).
 
-To post process to keep only data that was not filtered at the site level, you could use this command:
+```
+##FILTER=<ID=ExcessHet,Description="Site has excess het value larger than the threshold">
+##FILTER=<ID=NO_HQ_GENOTYPES,Description="Site has no high quality variant genotypes">
+##FILTER=<ID=EXCESS_ALLELES,Description="Site has an excess of alternate alleles based on the input threshold">
+##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=FT,Number=1,Type=String,Description="Genotype Filter Field">
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=RGQ,Number=1,Type=Integer,Description="Unconditional reference genotype confidence, encoded as a phred quality -10*log10 p(genotype call is wrong)">
+##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
+##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
+##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
+##INFO=<ID=AS_QUALapprox,Number=1,Type=String,Description="Allele-specific QUAL approximations">
+##INFO=<ID=QUALapprox,Number=1,Type=Integer,Description="Sum of PL[0] values; used to approximate the QUAL score">
+##COMMENT=<ID=high_CALIBRATION_SENSITIVITY_SNP,Description="Sample Genotype FT filter value indicating that the genotyped allele failed SNP model calibration sensitivity cutoff (0.997)">
+##COMMENT=<ID=high_CALIBRATION_SENSITIVITY_INDEL,Description="Sample Genotype FT filter value indicating that the genotyped allele failed INDEL model calibration sensitivity cutoff (0.99)">
+```
+
+Exomes will additionally have:
 
 ```
-bcftools view -f 'PASS,.' [YOUR VCF]
+##FILTER=<ID=OUTSIDE_OF_TARGETS,Description="Outside of sequencing target intervals">
```

Comment:
Ah interesting. I haven't been documenting everything in the header here, just the key information. I'm glad that is being written in the header personally, because it acts as provenance of what has been done to the VCF. We really should be adding a commandline entry with version information about what version of GVS was run or the tools within GVS or something to that effect if we aren't already. Something for the future.